README.md

```diff
@@ -1 +1,41 @@
-# capstone-project1
\ No newline at end of file
+# capstone-project1
+This is my first capstone project 
+
+***
+## Built With
+
+- html , css and javascript
+
+*** 
+## Getting Started
+**to implement and run this project you should have the** following prerequisites  
+
+## Prerequisites
+- Text editor
+- browser like crome
+
```

Comment:
- [ ] Please work on the depoyement since I can't see any link to live version to your website so kindly deploy it and add the link in to your README 👍🏻✅ since itis one of the requirements 

![image](https://user-images.githubusercontent.com/58553711/183259236-5378a0b8-fca7-40ed-ac6b-a98b744d4acd.png)